JS-Assignment-Operators.md

```diff
@@ -0,0 +1,75 @@
+# Javascript Assignment Operators
+
+
+## Introduction
+
+Assignment operators, as the name suggests, assigns (or re-assigns) values to a variable. While there are quite a few variations on the assignment operators, they all build off of the basic assignment operator.
```

Comment:
~~assigns (or re-assigns) values to a variable.~~ assign (or re-assign) values to a variable.